Gemfile.lock

```diff
@@ -124,7 +124,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```

Comment:
We use this for sidekiq-ent and pro. You can get the credentials for contribsys from https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/platform/engineering/sidekiq-enterprise-setup.md or just reset Gemfile.lock

Reply:
Resetting this will help get the Danger message for amount of changes in a better place :)

Reply:
@johnpaulashenfelter Thank you for the tip. I wondered about that. I have reverted the gemfile.lock back to it's original state.

Reply:
@johnpaulashenfelter Checking in with you on this. The PR says it's blocked by this requested change.